bloom/generators/debian/generator.py

```diff
@@ -511,6 +516,77 @@ def sanitize_package_name(name):
+            'exportvars': {'PYBUILD_INSTALL_ARGS':
+                           '--prefix="{0}" '
+                           '--install-lib="\$$base/lib/{{interpreter}}/site-packages" '
+                           .format(data['InstallationPrefix']), },
+        },
+        'cmake': {
+            'toplevel': '--buildsystem=cmake',
+            'autoconfigure': '-- -DCMAKE_INSTALL_PREFIX="{0}"'.format(data['InstallationPrefix']),
+            'depends': '${shlibs:Depends}',
+        },
+    }
+
+    build_type = package.get_build_type()
+
+    if build_type not in debhelper_options:
+        error('The build type `{build_type}`is not supported by this version of bloom.'
+              .format(build_type=build_type), exit=True)
+    data['debhelper_toplevel_options'] = debhelper_options[build_type].get('toplevel', '')
+    data['debhelper_autoconfigure_options'] = debhelper_options[build_type].get('autoconfigure', '')
+    data['DebhelperDepends'] = debhelper_options[build_type].get('depends', '')
+    data['exportvars'] = debhelper_options[build_type].get('exportvars', {})
+
+    # Build-type specific changes.
+    # This whole section is ripe for refator but it will do for now.
+    # Add build_types to the conditional block in alphabetical order.
+    if build_type == 'ament_cmake':
+        pass
+    elif build_type == 'ament_python':
+        # XXX inject build type specific dependencies.  For now this needs to be done
+        # after resolving rosdep keys as these are Debian platform-specific dependencies.
+        data['BuildDepends'] = list(set([
+                                        'dh-python',
+                                        'python3-all (>= 3.5)',
+                                        'python3-setuptools'
+                                        ] + data['BuildDepends']))
+        # Set `--install-scripts` unless it is specified in setup.cfg.
+        pass_install_scripts = True
+        package_path = os.path.abspath(os.path.dirname(package.filename))
+        setup_cfg_path = os.path.join(package_path, "setup.cfg")
+        if os.path.isfile(setup_cfg_path):
+            from configparser import SafeConfigParser
+            setup_cfg = SafeConfigParser()
+            setup_cfg.read([setup_cfg_path])
+            if (
+                setup_cfg.has_option('install', 'install-scripts') or
+                setup_cfg.has_option('install', 'install_scripts')
+            ):
+                pass_install_scripts = False
+        if pass_install_scripts:
+            data['exportvars']['PYBUILD_INSTALL_ARGS'] += ' --install-scripts="\$$base/bin" '
+    elif build_type == 'cmake':
+        pass
+    else:
+        # Even though we check supported build_types in debhelper options,
+        # preserve this so that forgetting to add a new build type to both the debhelper
+        # options and this conditional block will raise an error.
+        error('The build type `{build_type}`is not supported by this version of bloom.'
```

Comment:
nitpick, the standard style we use when quoting something in a message like this is to use `"` to capture the string, and `'` to quote the detail you want quoted. I'm not sure why this is prefered, but it's the common style. For example:

- https://github.com/ros-infrastructure/bloom/blob/master/bloom/config.py#L159
- https://github.com/ros-infrastructure/bloom/blob/master/bloom/git.py#L279
- https://github.com/ros-infrastructure/rosdistro/blob/master/src/rosdistro/source_build_file.py#L44

But of course there are always counterexamples:

- https://github.com/ros-infrastructure/catkin_pkg/blob/master/src/catkin_pkg/python_setup.py#L115

The important thing is that you're quoting it 👍, but if you want you can update it to be more consistent with the other code we have.